src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_java/java_logging.v2beta1.json.baseline

```diff
@@ -429,7 +433,8 @@ public class LoggingExample {
   public static void main(String args[]) throws IOException, GeneralSecurityException {
     // Required. The resource name of the project containing the metrics. Example:
     // `"projects/my-project-id"`.
-    String projectName = "projects/{MY-PROJECT}";  // TODO: Update placeholder value.
+    String projectName = "";  // TODO: Update placeholder value.
+    // ex: "projects/my-project"
```

Comment:
I'm fine with it - in logging we might just want to override the examples to be empty, seeing as they're part of the description.